modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/css/diagram.scss

```diff
@@ -114,3 +114,11 @@
 		}
 	}
 }
+
+.admin-tooltip {
+	background-color: #fff;
+	height: 300px;
+	position: absolute;
+	width: 400px;
+	z-index: 1;
```

Comment:
Please follow our z-index levels in DXP. https://github.com/liferay/clay/blob/87f7bfc1542a6b498db2073cbd5695a78f2eb377/packages/clay-css/src/scss/variables/_globals-z-index.scss#L7

Reply:
mmmm, can I add this link to this issue?


https://github.com/liferay/liferay-frontend-projects/issues/332 (z-index guidelines for DXP)

Reply:
@diegonvs yeah for sure

Reply:
thank you, but how can i add $zindex-tooltip? should i import it from where?